sdk/extensions/Microsoft.Azure.WebJobs.Extensions.Clients/README.md

```diff
@@ -0,0 +1,69 @@
+# Azure client library integration for WebJobs/Azure.Functions
+
+Microsoft.Extensions.Azure.Core provides shared primitives to integrate Azure clients with ASP.NET Core [dependency injection][dependency_injection] and [configuration][configuration] systems.
```

Comment:
Looks like this note wasn't updated. I was trying to figure out the purpose of this library.